src/openrct2-ui/windows/Footpath.cpp

```diff
@@ -1034,18 +1034,25 @@ class FootpathWindow final : public Window
 
         auto footpathPlaceAction = FootpathPlaceAction(
             footpathLoc, slope, type, gFootpathSelection.Railings, _footpathConstructDirection, constructFlags);
-        footpathPlaceAction.SetCallback([=](const GameAction* ga, const GameActions::Result* result) {
+
+        footpathPlaceAction.SetCallback([footpathLoc](const GameAction* ga, const GameActions::Result* result) {
+            auto* self = static_cast<FootpathWindow*>(WindowFindByClass(WindowClass::Footpath));
+            if (self == nullptr)
+            {
+                return;
```

Comment:
if only it were so easy... you still want to play the sound effect. I would have thought. The rest i think you don't need.